index.js

```diff
@@ -850,8 +857,14 @@ Command.prototype.description = function(str) {
  */
 
 Command.prototype.alias = function(alias) {
-  if (0 == arguments.length) return this._alias;
-  this._alias = alias;
+  var command = undefined;
+  if(this.commands.length == 0) {
+    command = this;
+  } else {
+    command = this.commands[this.commands.length - 1]
```

Comment:
there may some flaw here.  Some of the commands may don't have an alias.
I have made some change as below, test failed.
![image](https://cloud.githubusercontent.com/assets/681150/16550677/7fdebe3c-4173-11e6-9a9b-0f2cc7741ee0.png)